testing/cloud_layouts/vCloudDirector/Standard/configuration.tpl.yaml

```diff
@@ -0,0 +1,95 @@
+apiVersion: deckhouse.io/v1
+kind: ClusterConfiguration
+clusterType: Cloud
+cloud:
+  provider: VCD
+  prefix: '${PREFIX}'
+podSubnetCIDR: 10.111.0.0/16
+serviceSubnetCIDR: 10.222.0.0/16
+kubernetesVersion: '${KUBERNETES_VERSION}'
+defaultCRI: '${CRI}'
+clusterDomain: "cluster.local"
+---
+apiVersion: deckhouse.io/v1
+kind: InitConfiguration
+deckhouse:
+  imagesRepo: dev-registry.deckhouse.io/sys/deckhouse-oss
+  registryDockerCfg: '${DECKHOUSE_DOCKERCFG}'
+  devBranch: '${DEV_BRANCH}'
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: ModuleConfig
+metadata:
+  name: global
+spec:
+  enabled: true
+  settings:
+    modules:
+      publicDomainTemplate: "%s.k8s.smoke.flant.com"
+  version: 1
+
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: ModuleConfig
+metadata:
+  name: cni-cilium
+spec:
+  enabled: true
+
+---
+apiVersion: deckhouse.io/v1
+kind: VCDClusterConfiguration
+layout: Standard
+internalNetworkCIDR: 192.168.254.0/23 # 192.168.254.0/24 - dhctp, 192.168.255.0/24 - metalb
+masterNodeGroup:
+  instanceClass:
+    storageProfile: Fast vHDD 
+    sizingPolicy: ${VCD_ORG}-MSK
+    etcdDiskSizeGb: 10
+    rootDiskSizeGb: 40
+    template: ${VCD_ORG}/Ubuntu 24.04 Server (20240628)
+  replicas: 1
+nodeGroups:
+- instanceClass:
+    storageProfile:  Fast vHDD 
+    sizingPolicy: ${VCD_ORG}-MSK
+    rootDiskSizeGb: 40
+    template: ${VCD_ORG}/Ubuntu 24.04 Server (20240628)
+  name: system
+  replicas: 1
+  nodeTemplate:
+    labels:
+      node-role.deckhouse.io/system: ""
+    taints:
+    - effect: NoExecute
+      key: dedicated.deckhouse.io
+      value: system
+provider:
+  password: ${VCD_PASSWORD}
+  server: ${VCD_SERVER}
+  username: ${VCD_USERNAME}
+  insecure: true
+organization: ${VCD_ORG}
+virtualApplicationName: deckhouse-e2e 
+virtualDataCenter: ${VCD_ORG}-MSK1-S1-vDC2
+mainNetwork:  deckhouse-e2e
+sshPublicKey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDSNdUmV2ekit0rFrQE9IoRsVqKTJfR8h+skMYjXHBv/nJN6J2eBvQlebnhfZngxTvHYYxl0XeRu3KEz5v23gIidT21o9x0+tD4b2PcyZ24o64GwnF/oFnQ9mYBJDRisZNdXYPadTp/RafQ0qNUX/6h8vZYlSPM77dhW7Oyf6hcbaniAmOD30bO89UM//VHbllGgfhlIbU382/EnPOfGvAHReATADBBHmxxtTCLbu48rN35DlOtMgPob3ZwOsJI3keRrIZOf5qxeF3VB0Ox4inoR6PUzWMFLCJyIMp7hzY+JLakO4dqfvRJZjgTZHQUvjDs+aeUcH8tD4Wd5NDzmxnHLtJup0lkHkqgjo6vqWIcQeDXuXsk3+YGw0PwMpwO2HMVPs2SnfT6cZ+Mo6Dmq0t1EjtSBXLMe5C5aac5w6NrXuypRQDoce7p3uZP2TVsxmpyvkd6RyiWr+wuOOB3h/k8q+kRh4LKzivJMEkZoZeCxkJiIWDknxEAU1sl25W4hEU=
+
+---
+apiVersion: deckhouse.io/v1alpha1
+kind: ModuleConfig
+metadata:
+  name: flant-integration
+spec:
+  enabled: false
+
+---
+apiVersion: deckhouse.io/v1
```

Comment:
Please move to resources.yaml

Reply:
I can't, because I'm using templating here.